katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
```

Comment:
There are only three places we backup? I thought we also backed up configuration?